### ISSUE TITLE
Can not update SHA1 within a module

### ISSUE THREAD

**Issue by dadoonet** [elastic/elasticsearch#15224]
I'm trying to upgrade some libs in `plugins/discovery-azure`. So I need to update the `SHA1` files.

```sh
cd plugins/discovery-azure
gradle updateSHAs
```

It gives:

```
...
=======================================
Elasticsearch Build Hamster says Hello!
=======================================
  Gradle Version        : 2.8
  OS Info               : Mac OS X 10.11.1 (x86_64)
  JDK Version           : Oracle Corporation 1.8.0_60 [Java HotSpot(TM) 64-Bit Server VM 25.60-b23]
:plugins:discovery-azure:updateShas FAILED

FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':plugins:discovery-azure:updateShas'.
> Could not find property 'SHA_EXTENSION' on task ':plugins:discovery-azure:updateShas'.
```

**Comment by dadoonet** [comment]
Strange. I tried from the root project dir and it failed with:

```
:core:updateShas FAILED

FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':core:updateShas'.
> java.io.FileNotFoundException: /Users/dpilato/Documents/Elasticsearch/dev/es-gradle/elasticsearch/core/licenses
```

@rjernst Any idea?

**Comment by rjernst** [comment]
Sorry @dadoonet! I opened a PR to fix these problems here: #15226
